### PR TITLE
Bsbport debug output and fix

### PR DESCRIPTION
**Pull request by AnDann**
readabilty of debug output improved
two small fixes

### CODE REVIEW

File: protocols/bsbport/bsbport_ecmd.c [hunk 1]
```diff
@@ -33,6 +33,12 @@
 #include "protocols/bsbport/bsbport_tx.h"
 #include "protocols/ecmd/ecmd-base.h"
 
+#ifdef DEBUG_BSBPORT_ECMD
+#define BSBPORT_DEBUG(s, args...) debug_printf("BSB " s "\n", ## args);
```
Comment (eku) [inline protocols/bsbport/bsbport_ecmd.c]: Why are we using these old-fashioned conctruct? SInce we compile at least with C99 why not use variadic macros?

**Comment by AnDann** [comment]
I found that somewere else in the ethersex code. 
Do you mean something like this:
```c
#define BSBPORT_DEBUG(s, ...) debug_printf("BSB " s "\n", __VA_ARGS__);
```

**Comment by eku** [comment]
Yes, this is a variadic macro.

**Comment by AnDann** [comment]
If the __VA_ARGS__ argument is empty this will produce a syntax error

**Comment by AnDann** [comment]
Did change all debug macros

**Pull request merged**